Lesson3.ipynb

```diff
@@ -29,6 +32,440 @@
     "\n",
     "Select **Lesson3.ipynb**"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loop\n",
+    "### What is a loop?\n",
+    "A loop is a sequence of statements which is specified once but which may be carried out several times in succession (Wikipedia).\n",
+    "\n",
+    "In other words, instead of giving the computer instructions like below\n",
+    "> Hi computer, please print 'Hello, World!'\n",
+    "\n",
+    "> please print another 'Hello, World!'\n",
+    "\n",
+    "> ... (continue to ask the computer to print)\n",
+    "\n",
+    "> print the 10th 'Hello, world!'\n",
+    "\n",
+    "we want to give this computer this short instruction only\n",
+    "> Hi computer, please print 'Hello, World!' 10 times\n",
+    "\n",
+    "\n",
+    "In code form:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Without loop\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')\n",
+    "print('Hello, World!')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# With loop\n",
+    "for i in range(10):\n",
```

Comment:
Is it worth explaining that range(n) in Python means 0, 1, 2... n-1 since Python is 0-index based?